.github/workflows/deploy-dotorg.yml

```diff
@@ -12,12 +12,28 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
+
+      - name: Setup Node
+        uses: actions/setup-node@v3.5.0
+        with:
+          node-version-file: ".nvmrc"
+          cache: npm
+
+      - name: Install Node production dependencies
+        run: npm ci --production
+        env:
+          CI: true
+
+      - name: Add Partytown library files to release
+        run: npm run partytown
+
```

Comment:
Having a build process is also needed for Image Loading Optimization. Referenced in #869 pointing to https://github.com/WordPress/performance/pull/876#discussion_r1380553929. We need to use a local copy of web-vitals.js.

Reply:
@westonruter @adamsilverstein Added a CLI command called `copy-vendor-libs` in https://github.com/WordPress/performance/pull/556/commits/e6a51cecf3e716b2a263838b970e4b6574bebeb5.

Tasks it will perform:
- Copy vendor files from source to destination based on `copy-vendor-libs` config in `package.json`.
- Copy files from source to destination based on command args i.e. `./bin/plugin/cli.js copy-vendor-libs --source foo/bar/lib --destination modules/module`
- Run after executing the `npm install` command (prepare hook).
- Run with the `build-plugins` command, just before creating the standalone plugin builds.

Reply:
We have removed the above logic and now using Webpack to copy the assets. See https://github.com/WordPress/performance/pull/556/commits/5dfcfdfee96f090cd18889dbc6bc9b3a68d875ac and https://github.com/WordPress/performance/pull/953